test/unit/specs/components/staking/__snapshots__/PageStaking.spec.js.snap

```diff
@@ -21,107 +21,9 @@ exports[`PageStaking has the expected html structure 1`] = `
         <div
           class="tm-page-header-body"
         >
-          <div
-            class="header-balance"
-          >
-            <div
-              class="top"
-            >
-              <div
-                class="icon-container"
-              >
-                <img
-                  class="icon"
-                  src="~assets/images/cosmos-logo.png"
-                />
-              </div>
-               
-              <div
-                class="total-atoms top-section"
-              >
-                <h3>
-                  Total STAKE
-                </h3>
-                 
-                <h2
-                  class="total-atoms__value"
-                >
-                  1,351.0000…
-                </h2>
-              </div>
-               
-              <div
-                class="unbonded-atoms top-section"
-              >
-                <h3>
-                  Available STAKE
-                </h3>
-                 
-                <h2>
-                  1,337.0000…
-                </h2>
-              </div>
-            </div>
-             
-            <div
-              class="short-bech32"
-            >
-              <div
-                class="address"
-                id="address"
-              >
-                
-    cosmos…xxn9
-  
-              </div>
-               
-              <div
-                class="copied"
-              >
-                <i
-                  class="material-icons"
-                >
-                  check
-                </i>
-                <span>
-                  Copied
-                </span>
-              </div>
-            </div>
-             
-            <div
-              class="tabs"
-            >
-              <div
-                class="tab"
-              >
-                <router-link-stub
-                  to="[object Object]"
-                >
-                  My Delegations
-                </router-link-stub>
-              </div>
-              <div
-                class="tab"
-              >
-                <router-link-stub
-                  to="[object Object]"
-                >
-                  Validators
-                </router-link-stub>
-              </div>
-              <div
-                class="tab"
-              >
-                <router-link-stub
-                  to="[object Object]"
-                >
-                  Parameters
-                </router-link-stub>
-              </div>
-            </div>
-             
-          </div>
+          <tm-balance-stub
+            tabs="[object Object],[object Object],[object Object]"
```

Comment:
?

Reply:
shallowMount produces this output. it doesn't serialize objects.